### DISCUSSION TITLE
Textbox in likert plugin

### DISCUSSION THREAD

**Discussion by SophiaAdamWangels** [jspsych/jsPsych#1933] [Q&A]
Hi all, 
I am new to jsPsych (and coding in general) and would like to ask if it is possible to include a fill-in textbox in the likert survey format. I haven't been able to figure out a way. I would like it to look something like in the screenshot I am attaching. 

Can someone tell me if that would work at all and if so, how to go about it? I would appreciate it!
![Screenshot 2021-06-27 at 22 21 47](https://user-images.githubusercontent.com/86537423/123561356-321a6400-d7a8-11eb-90a1-2882021531ad.png)

**Comment by GEJ1** [comment]
Hi @SophiaAdamWangels 

If i'm understanding correctly you can make this type of experments using [jspsych-survey-multi-choice plugin](https://www.jspsych.org/plugins/jspsych-survey-multi-choice/).

You can find an use case in [jsPsych/examples/jspsych-survey-multi-choice.html
](https://github.com/jspsych/jsPsych/blob/master/examples/jspsych-survey-multi-choice.html)
![image](https://user-images.githubusercontent.com/8774034/123714842-a0256080-d866-11eb-976f-798d14f3385f.png)

Edit: I just realized that the question was for the fill-in textbox. In that case maybe the [jspsych-survey-html-form plugin](https://www.jspsych.org/plugins/jspsych-survey-html-form/) it's a better option.

Best!

Gustavo.

**Reply by SophiaAdamWangels** [reply]
Thanks for your advice Gustavo, I will try!

**Comment by becky-gilbert** [comment]
HI @SophiaAdamWangels, I agree with Gustavo, the survey-html-form plugin might be your best bet. Another option is to modify the survey-likert plugin file so that it (1) includes a parameter for adding a textbox to each question, and (2) saves the textbox response to the trial data. Please let us know if you need any further guidance with either of these options.
(And FYI, we're aware that the current survey-* plugins aren't very flexible, so we're planning to re-vamp them in the near future!)
